brian2/core/network.py

```diff
@@ -52,6 +52,7 @@
 
 
 def _format_time(time_in_s):
+    print(time_in_s,"hereeeeeee")
```

Comment:
Quite sure that shouldn't be here anymore :smile:

Reply:
Yes. Really sorry for this mistake.